lib/matplotlib/streamplot.py

```diff
@@ -20,61 +20,64 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
                cmap=None, norm=None, arrowsize=1, arrowstyle='-|>',
                minlength=0.1, transform=None, zorder=None, start_points=None,
                maxlength=4.0, integration_direction='both'):
-    """Draw streamlines of a vector flow.
-
-    *x*, *y* : 1d arrays
-        an *evenly spaced* grid.
-    *u*, *v* : 2d arrays
-        x and y-velocities. Number of rows should match length of y, and
-        the number of columns should match x.
-    *density* : float or 2-tuple
-        Controls the closeness of streamlines. When `density = 1`, the domain
+    """
+    Draw streamlines of a vector flow.
+
+    Parameters
+    ----------
+    x, y : 1d arrays
+        An evenly spaced grid.
+    u, v : 2d arrays
+        *x* and *y*-velocities. Number of rows should match length of *y*, and
+        the number of columns should match *x*.
+    density : float or 2-tuple
+        Controls the closeness of streamlines. When ``density = 1``, the domain
         is divided into a 30x30 grid---*density* linearly scales this grid.
         Each cell in the grid can have, at most, one traversing streamline.
         For different densities in each direction, use [density_x, density_y].
-    *linewidth* : numeric or 2d array
-        vary linewidth when given a 2d array with the same shape as velocities.
-    *color* : matplotlib color code, or 2d array
+    linewidth : numeric or 2d array
+        Vary linewidth when given a 2d array with the same shape as velocities.
+    color : matplotlib color code, or 2d array
         Streamline color. When given an array with the same shape as
         velocities, *color* values are converted to colors using *cmap*.
-    *cmap* : :class:`~matplotlib.colors.Colormap`
+    cmap : :class:`~matplotlib.colors.Colormap`
         Colormap used to plot streamlines and arrows. Only necessary when using
         an array input for *color*.
-    *norm* : :class:`~matplotlib.colors.Normalize`
-        Normalize object used to scale luminance data to 0, 1. If None, stretch
-        (min, max) to (0, 1). Only necessary when *color* is an array.
-    *arrowsize* : float
+    norm : :class:`~matplotlib.colors.Normalize`
+        Normalize object used to scale luminance data to 0, 1. If ``None``,
```

Comment:
Personally, I'd go for `*None*` instead. But we don't have a common practice defined for these and both variants do currently occur in the code.

Reply:
I use double ticks since `None` is technically a snippet of code. I thought stars were reserved for keyword arguments?

Reply:
- https://devguide.python.org/documenting/ recommends code-format
- [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) and [sphinx](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html) do not use any highlighting
- I don't remember where, but I've seen italic format as well.

There is no universal standard but code format is fine. We might want to agree on one style but that's something for another issue.